apps/client/src/hoc/withChecker.js

```diff
@@ -22,42 +22,54 @@ const withChecker = (Component) =>
 
     if (sttrFile) {
       useEffect(() => {
-        if (!checker && !error) {
-          fetch(`${window.location.origin}/sttr/${dir}/${sttrFile}`)
-            .then((response) => response.json())
-            .then((json) => {
-              const newChecker = getChecker(json);
-              // Find if we have missing data needs
-              if (address) {
-                newChecker.autofill(autofillResolvers, { address });
-              }
-              const unfulfilledDataNeed = newChecker.getAutofillDataNeeds(
-                autofillMap,
-                true
-              )[0];
-
-              if (sessionContext[slug]?.answers && !unfulfilledDataNeed) {
-                newChecker.setQuestionAnswers(sessionContext[slug].answers);
-                // @TODO In case of reload, rewind to the current question
-              }
-              // Store the entire `sttr-checker` in React Context
-              checkerContext.checker = newChecker;
-              setChecker(newChecker);
-            })
-            .catch((e) => {
-              setError(e);
-            });
-        }
+        initChecker();
```

Comment:
Now the initChecker() is above the function declaration, shall we place this below?